src/pybreaker.py

```diff
@@ -362,6 +366,139 @@ def opened_at(self, datetime):
         """
         self._opened_at = datetime
 
+import logging
```

Comment:
@phillbaker sorry for the delay!

Could you move this `import` to the beginning of the file, next to the other ones?